analysisCode/src/EventLoop.cpp

```diff
@@ -1,51 +1,59 @@
-/// Fastjet includes
 #include <fastjet/ClusterSequence.hh>
 #include <fastjet/Selector.hh>
 #include <fastjet/contrib/SoftDrop.hh>
+#include <iostream>
 
-/// Root includes
 #include <TFile.h>
 #include <TTree.h>
 #include <TSystem.h>
-
-/// EIC smear includes
 #include <eicsmear/erhic/EventPythia.h>
 #include <eicsmear/smear/EventS.h>
-
-/// Local includes
 #include "include/TruthEvent.h"
 #include "include/SmearedEvent.h"
 #include "include/JetDef.h"
 #include "include/SoftDropJetDef.h"
 
-#include <iostream>
+using namespace std;
+using namespace fastjet;
 
-int main(int argc, char **argv)
+int main()
 {
   /// Collect arguments
-  std::string mcFile = argv[1];
-  std::string smearedFile = argv[2];
-
-  TFile mc(mcFile.c_str());
+  //std::string mcFile = argv[1];
+  //std::string smearedFile = argv[2];
+  TFile mc("../truth.root");
   TTree *mctree = (TTree*)mc.Get("EICTree");
+  mctree->AddFriend("Smeared","../smeared.root");
+
+ //TFile mc(mcFile.c_str());
+ // TTree *mctree = (TTree*)mc.Get("EICTree");
+
+  // Jet Def Stuff 
+  double R = 0.7;
+  JetDef jetDef(antikt_algorithm, R);
+
+  /// soft drop conditions
+  double zcut = 0.1;
+  double beta = 0;
+  SoftDropJetDef softdrop_def(beta, zcut, R);
   
   TruthEvent event;
   SmearedEvent smearevent;
-  JetDef jetdef;
-  SoftDropJetDef sd(0.1, 2, 0.5);
   
-  mctree->AddFriend("Smeared", smearedFile.c_str());
+  //mctree->AddFriend("Smeared", smearedFile.c_str());
   erhic::EventPythia* truthEvent(NULL);
   Smear::Event* smearEvent(NULL);
 
   mctree->SetBranchAddress("event", &truthEvent);
   mctree->SetBranchAddress("eventS", &smearEvent);
 
-  for(int event = 0; event < mctree->GetEntries(); ++event)
+  for (int ev = 0; ev < 10; ev++)
```

Comment:
Is there a reason this was changed to 10 instead of the entire tree?

Reply:
This was also just for testing, I did not mean to include this change in this pull request.